cmd/askd/handlers/form_submission.go

```diff
@@ -253,3 +307,25 @@ func (formSubmissionHandle) RemoveFlag(c *app.Context) error {
 
 	return nil
 }
+
+// encodeCSV gets all the submissions and encode them into a CSV
+func encodeCSV(s []submission.Submission) ([]byte, error) {
+
+	var buf bytes.Buffer
+	w := csv.NewWriter(&buf)
+	if err := w.Write(s[0].GetQuestions()); err != nil {
```

Comment:
Yes.  In this case we may have replies for questions that do not exist anymore. Not sure if we keep adding questions to the headers and then leave a blank cell for the rest.